.github/workflows/ci.yml

```diff
@@ -24,17 +24,11 @@ jobs:
         aws-region: eu-west-1
     - uses: guardian/actions-setup-node@v2.4.1
     - name: Set up JDK
-      uses: actions/setup-java@v2
+      uses: actions/setup-java@v4
       with:
-        java-version: 8
-        distribution: adopt
-    - name: Cache SBT
-      uses: actions/cache@v2
-      with:
-        path: |
-          ~/.ivy2/cache
-          ~/.sbt
-        key: ${{ runner.os }}-sbt-${{ hashFiles('**/build.sbt') }}
```

Comment:
The `setup-java` GHA now has built-in support for `sbt` caching (added with https://github.com/actions/setup-java/pull/302) so we don't need to try to provide our own definition for caching sbt resources.